### PR TITLE
feat(Features) : Add dashboard visualization to graphql api

### PR DESCRIPTION
**Pull request by cheikhgwane**

### CODE REVIEW

**Comment by qgerome** [comment]
@cheikhgwane Je pense que tu as encore des soucis avec ta config lint

**Comment by cheikhgwane** [comment]
> @cheikhgwane Je pense que tu as encore des soucis avec ta config lint

Je suis en train de voir ça.

File: hexa/visualizations/admin.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class IndexAdmin(admin.ModelAdmin):
     list_filter = ("content_type",)
     search_fields = ("label", "content")
 
-    def app_label(self, obj):
+    def app_id(self, obj):
```
Comment (qgerome) [inline hexa/visualizations/admin.py:27]: Why did you change `app_label` to `app_id` ?

File: hexa/visualizations/schema.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        .order_by("-updated_at")
+    )
+
+    return result_page(queryset=queryset, page=page, per_page=per_page)
+
+
+@dashboard_queries.field("externalDashboard")
+def resolve_external_dashboard(_, info, **kwargs):
+    try:
+        request = info.context["request"]
```
Comment (qgerome) [inline hexa/visualizations/schema.py:63]: This can be above the `try..catch` as it is not what you'd like to test

**Review by qgerome — CHANGES_REQUESTED** [review]

File: hexa/visualizations/admin.py (hunk 1 above)
Comment (cheikhgwane) [reply hexa/visualizations/admin.py:27]: It was  a test in order to make the label clickable on the admin. But I'll remove it from now.

File: hexa/visualizations/schema.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    return reverse(
+        "visualizations:dashboard_image", kwargs={"dashboard_id": dashboard.id}
+    )
+
+
+dashboard_queries = QueryType()
+
+
+@dashboard_queries.field("externalDashboards")
+def resolve_external_dashboards(_, info, page=1, per_page=15):
```
Comment (qgerome) [inline hexa/visualizations/schema.py:49]:
```suggestion
def resolve_external_dashboards(_, info, page=1, perPage=15):
```

Comment (cheikhgwane) [reply hexa/visualizations/schema.py]: It's updated

**Comment by qgerome** [comment]
It would be nice to have a test where there are more than two pages of items

**Review by qgerome — APPROVED** [review]
